### ISSUE TITLE
scope is insufficient

### ISSUE THREAD

**Issue by p2pow350** [TurboGit/hubicfuse#43]
Hi Turbo!
I found that other user on forum are facing same problem…
Can you help me to solve this issue?
I'm really not able to access my folders with Ubuntu 14 and Centos 6. It always say to me that the scope is insufficient!
This could be a permission problem, but once I've created the API Key and Secret under the Hubic page, what else I've to do?
Seems that token parameter is missing, but the command "hubic_token" is not generating this value.
Here is what I'm getting.

Merçi beaucoup!
Max

hubicfuse -d /mnt/ -o noauto_cache,sync_read,allow_other

!!! Authenticating... (client_id = 'api_hubic_xxxxxxxxxxxxxxxxxxxxxxxxxxxx')
!!! HUBIC TOKEN_URL result: '{"expires_in":21600,"access_token":"xxkahsdfjhaldjskfladhsfljkahsdlfhaklsdfhlakdhsfkljahsdlf","token_type":"Bearer"}'

!!! HUBIC Access token: akjsdflòkjasdòfjaòlsdfjòaklsdfòans.mz,nxcvòoijqwkònasòdkljaòj

!!! HUBIC Token type  : Bearer

!!! HUBIC Expire in   : 21600

!!! CRED_URL result: '{
    "error":"insufficient_scope",
    "error_description":"scope is insufficient"
}
'

!!! HUBIC cannot get json field 'token'

Failed to authenticate.

**Comment by TurboGit** [comment]
You really need to get a token from hubic_token or from the Web interface.

What is the content of your .hubifuse (was .cloudfuse) ?

You should have three parameters:

   client_id=api_hubic_...
   client_secret=...
   refresh_token=...

**Comment by TurboGit** [comment]
Also, depending on the way you run mount the .hubicfuse file must be on your $HOME or in /root.

**Comment by p2pow350** [comment]
Hi Pascal!
to avoid any problems regarding permissions I'm trying to mount my Hubic folders with user "root" on my both machines and also I got the changes in configuration, so I'm running command hubic_token providing my account info an then I'm saving the file into /root/.hubicfuse.

But the problem is still there…I've even deleted my "application" and created another into the developer section of the Hubic configuration page but nothing changes..

And also, I was making some tests today within the api. page but on some API procedures I'm getting same error, so I suspect I'm missing something on Hubic page.

BTW, thank for your support
Cheers
Max

**Comment by TurboGit** [comment]
So this does not seems an hubicfuse issue. Not sure what it could be!!! A special character into the password? Or the redirected URL? Are you using localhost? I don't rememeber all the details I've registered a very long time ago...

**Comment by TurboGit** [comment]
And (just in case) you are using the latest hubicfuse source, right?

**Comment by p2pow350** [comment]
Yes, I've downloaded latest and it doesn't work….it worked on previous version where the configfile was .cloudfuse…sigh!
My redirect host is https://www.google.com, could be?

**Comment by TurboGit** [comment]
The change of authentication is due to OVH so it was a good time to move from .cloudfuse to .hubicfuse. In any case the previous authentication values are not usable anymore.

The www.google.com could be the issue, yes.

Maybe there is people having the same issue on the OVH forum?

**Comment by p2pow350** [comment]
I've just deleted old application credentials and just created one new using http://localhost/ as redirect host, but still same problem…

I wonder why it's asking me for field token…..this parameter is not in .hubicfuse file, maybe it's this one?

!!! HUBIC cannot get json field 'token'

**Comment by TurboGit** [comment]
Hum, wait do you enter "yes" explicitly to permissions question when running hubic_token?

**Comment by TurboGit** [comment]
Not, yes sorry but the actual permission in '(' ')'. You must enter the text inside the parentésis.

**Comment by p2pow350** [comment]
Oh my goodness!!
I'm a skilled developer with 15 years of experience in computer programming and system administration and I forgot to type values written within parenthesis as I was sure they were default values……
What a stupid I'm am!! After I typed values as suggested all values are generated and that is, now I can mount my hubic account!!

Just to understand how I feel, I spent years laughing of some customers for which I gave support with something like that images and now it happens to me! LOL!
http://img4.wikia.nocookie.net/__cb20051107140117/uncyclopedia/images/b/b7/Press_any_key.jpg

Thank you for this great plugin and for this support!
